pkg/tools/etcd_helper_watch_test.go

```diff
@@ -205,7 +217,7 @@ func TestWatchEtcdError(t *testing.T) {
 	fakeClient := NewFakeEtcdClient(t)
 	fakeClient.expectNotFoundGetSet["/some/key"] = struct{}{}
 	fakeClient.WatchImmediateError = fmt.Errorf("immediate error")
-	h := EtcdHelper{fakeClient, codec, versioner}
+	h := NewEtcdHelper(fakeClient, codec)
```

Comment:
I'm a little worried about removing the versioner setting here-- it means that this test will test inconsistent sets of things if someone changes the "versioner" at the top of the file OR if someone changes the versioner that NewEtcdHelper uses. It's not too likely to change so I'm not too worried, just vaguely uneasy.